### PR TITLE
Adding EBEN (SEP-20)

### PR DESCRIPTION
**Pull request by BenTokenFinance**
### Token Details
- **Token symbol:**  EBEN
- **Token name:**  Green Ben
- **Wallet address used to provide initial pool liquidity:**   0x989947bFe861a5B976b83E872Fc2e5A5156103b8  (Dev Wallet)

### Checklist
- [x] Filled out token details above
- [x] Added token info to tokens.json
- [x] Added token icon to /icons

### CODE REVIEW

**Review by allisonle — APPROVED** [review]

**Comment by allisonle** [comment]
Hi @BenTokenFinance, the BCH/EBEN pair is now ready to be funded by the provided address on https://verse.bitcoin.com/!

**Comment by BenTokenFinance** [comment]
Thank you! Pair funded.

**Comment by Meems2020** [comment]
I'm new to this and really am having a hard time understanding exactly what it is I should do on here like how do I get paid how do I make money using this site trying to understand my new way of life...thanks

**Comment by ashnbill** [comment]
Yes

File: config/tokens.json
Comment (walker112171) [inline config/tokens.json:1]: Ty

**Pull request merged**